.github/workflows/test.yaml

```diff
@@ -12,13 +19,19 @@ jobs:
           - beta
           - nightly
     steps:
-      - uses: actions/checkout@v1
-      - name: Install Rust
-        uses: actions-rs/toolchain@v1
+      - uses: actions/checkout@v3
+      - uses: dtolnay/rust-toolchain@master
         with:
           toolchain: ${{ matrix.rust }}
-          override: true
           components: rustfmt
       - name: Run tests
-        run: ./ci.sh
```

Comment:
Can we keep those script as separated files? They are helpful when locally testing without CI.

Reply:
NB: I often use https://github.com/nektos/act to test my Github Action pipelines.